x/oracle/proposal_handler.go

```diff
@@ -0,0 +1,37 @@
+package oracle
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/medibloc/panacea-core/v2/x/oracle/keeper"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+)
+
+func NewOracleProposalHandler(k keeper.Keeper) govtypes.Handler {
+	return func(ctx sdk.Context, content govtypes.Content) error {
+		switch c := content.(type) {
+		case *types.OracleUpgradeProposal:
+			return handlerOracleUpgradeProposal(ctx, k, c)
+		default:
+			return sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "unrecognized oracle upgrade proposal content type: %T", c)
+		}
+
+	}
+}
+
+func handlerOracleUpgradeProposal(ctx sdk.Context, k keeper.Keeper, p *types.OracleUpgradeProposal) error {
```

Comment:
Wait. When this proposal is passed, the uniqueID in `x/oracle` module params should be changed at target height. Then, I think we need to add the logic to `x/oracle` end blocker. Am I right?

Reply:
Yes, that's right. Development for endblocker is going on in another PR.

Reply:
When the upgrade height is reached, the params of x/oracle will be modified and the contents of OracleUpgradeInfo will be cleared.